etc/adminhtml/system.xml

```diff
@@ -95,6 +95,11 @@
                         <config_path>payment/boltpay/order_management</config_path>
                         <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     </field>
+                    <field id="always_present_checkout" translate="label" type="select" sortOrder="83" showInDefault="0" showInWebsite="0" showInStore="0">
```

Comment:
Do you want to hide this setting?

Reply:
I want to show it. This way a merchant who we enable through feature switch can also turn it off if they want.

Reply:
I see. Then we need set parameters to:
**showInDefault="1" showInWebsite="1" showInStore="0"**